src/sentry/monitors/system_incidents.py

```diff
@@ -42,6 +49,10 @@
 # We record 30 days worth of historical data for each minute of check-ins.
 MONITOR_VOLUME_RETENTION = timedelta(days=30)
```

Comment:
I wonder if it's worth just setting this a bit higher in case we decide we want the older data